frontend/package.json

```diff
@@ -33,6 +35,7 @@
     "react-router-dom": "^6.26.0",
     "react-slick": "^0.30.2",
     "styled-components": "^6.1.12",
+    "use-react-router-breadcrumbs": "^4.0.1",
```

Comment:
this is an outdated package, last update on it was 2 years ago, and they mentioned that v4 of react router is supporting breadcrumbs natively